DragAndDrop/DraggableView.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  Drag.swift
+//  DragAndDrop
+//
+//  Created by Stanley Chiang on 3/6/16.
+//  Copyright © 2016 SantaClaraiOSConnect. All rights reserved.
+//
+
+import UIKit
+
+protocol draggableViewDelegate {
+    func didDragToTrash() -> Bool
+}
+
+class DraggableView: UIView {
+    
+    var panGestureRecognizer:UIPanGestureRecognizer!
+    var initialPosition:CGPoint!
+    
+    var delegate:draggableViewDelegate!
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        initialPosition = frame.origin
```

Comment:
Perhaps move this saving of the `initialPosition` to where the PanGesture starts?

This is a win for me on 2 points:
* Moves stuff out of initializer
* Moves the variable initial value closer to where it is actually used.